extensions/languagetool/package.json

```diff
@@ -0,0 +1,455 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "languagetool",
+  "title": "LanguageTool",
+  "description": "Integrates LanguageTool spell-checking directly into Raycast",
+  "icon": "extension-icon.png",
+  "author": "marvinvr",
+  "type": "module",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "check-spelling",
+      "title": "Check Spelling",
+      "subtitle": "Use LanguageTool to check spelling",
+      "description": "Check spelling of selected text or clipboard content", "Description": "Check spelling of selected text or clipboard content",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "endpoint",
+      "title": "API Endpoint",
+      "description": "The API endpoint to use",
+      "type": "textfield",
+      "required": true,
+      "default": "https://api.languagetool.org/v2/check"
+    },
```

Comment:
Is this `endpoint` preference option needed? Does LanguageTools support any other enpoints?